tests/functional/u/used/used_before_assignment_typing.py

```diff
@@ -26,11 +35,13 @@
     import types
     import zoneinfo
 elif input():
-    import calendar, bisect  # pylint: disable=multiple-imports
+    import calendar, bisect, pathlib  # pylint: disable=multiple-imports
     if input() + 1:
         import heapq
     else:
         import heapq
+    if TYPE_CHECKING:
+        import math
```

Comment:
Same, this doesn't seem to be targeted by any tests.

Reply:
Similar to above, the intention was to ensure that `math` is not defined by line 44 and `used-before-assignment` is raised.